apps/editor/src/__test__/wwCommand.spec.ts

```diff
@@ -0,0 +1,420 @@
+
+  function setTextToEditor(text: string) {
+    const { state, dispatch } = wwe.view;
+    const { tr, doc } = state;
+    const lines = text.split('\n');
+    const node = lines.map(lineText =>
+      wwe.schema.nodes.paragraph.create(null, wwe.schema.text(lineText))
+    );
+
+    dispatch(tr.replaceWith(0, doc.content.size, node));
+  }
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+
+    em = new EventEmitter();
+    wwe = new WysiwygEditor(container, em);
+    cmd = new CommandManager(em, {}, wwe.commands);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+  });
+
+  describe('heading command', () => {
+    it('should add empty heading element', () => {
+      cmd.exec('wysiwyg', 'heading', { level: 1 });
+
+      expect(wwe.getHTML()).toBe('<h1><br></h1>');
+    });
+
+    it('should add heading element to selection', () => {
+      setTextToEditor('foo');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'heading', { level: 2 });
+
+      expect(wwe.getHTML()).toBe('<h2>foo</h2>');
+    });
+
+    it('should change heading element by level', () => {
+      setTextToEditor('foo');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'heading', { level: 3 });
+
+      expect(wwe.getHTML()).toBe('<h3>foo</h3>');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'heading', { level: 4 });
+
+      expect(wwe.getHTML()).toBe('<h4>foo</h4>');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'heading', { level: 5 });
+
+      expect(wwe.getHTML()).toBe('<h5>foo</h5>');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'heading', { level: 6 });
+
+      expect(wwe.getHTML()).toBe('<h6>foo</h6>');
+    });
+  });
+
+  describe('hr command', () => {
+    it('should add hr element with wrapping div element', () => {
+      cmd.exec('wysiwyg', 'hr');
+
+      expect(wwe.getHTML()).toBe('<div><hr></div>');
+    });
+
+    it('should change hr element to selection', () => {
+      setTextToEditor('foo');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'hr');
+
+      expect(wwe.getHTML()).toBe('<div><hr></div>');
+    });
+  });
+
+  describe('blockQuote command', () => {
+    it('should add blockquote element including empty paragraph', () => {
+      cmd.exec('wysiwyg', 'blockQuote');
+
+      expect(wwe.getHTML()).toBe('<blockquote><p><br></p></blockquote>');
+    });
+
+    it('should change blockquote element to selection', () => {
+      setTextToEditor('foo');
+
+      cmd.exec('wysiwyg', 'selectAll');
+      cmd.exec('wysiwyg', 'blockQuote');
+
+      expect(wwe.getHTML()).toBe('<blockquote><p>foo</p></blockquote>');
+    });
+
+    // @TODO In the current specification, when a command is
+    // executed in a block quote element, the element disappears.
+    // Discuss specification maintaining.
```

Comment:
> 논의 : v2에서는 `blockQuote` 커맨드 실행 시 토글로 동작하나 프로즈미러에서는 인용구가 하위에 추가되도록 동작한다. 인용구는 블록 노드이기 때문에 커맨드를 실행했을 때 토글이 아니라 다른 블록 노드와 동일하게 노드가 추가되도록 변경되는 것이 맞지 않을까?

### AS-IS (v2)
![v2](https://user-images.githubusercontent.com/18183560/95714692-8d9cd400-0ca3-11eb-89a8-bc73f1adc809.gif)

### TO-BE (v3)
![v3](https://user-images.githubusercontent.com/18183560/95714706-92fa1e80-0ca3-11eb-8a53-e7d21f99c603.gif)